permission.c

```diff
@@ -220,7 +212,7 @@ static const luaL_Reg lib[] =
 };
 
 LUALIB_API int
-luaopen_cujo_permission(lua_State *L)
+luaopen_rabid_permission(lua_State *L)
```

Comment:
Let's keep `cujo` namespace since this module will not be embedded in Rabid anymore, but distributed as an independent module.
Just keep the license change.